app/controllers/api/v1/me_controller.rb

```diff
@@ -0,0 +1,20 @@
+module Api
+  module V1
+    class MeController < ApplicationController
```

Comment:
Why don't you just use the UsersController instead of create a new controller? Also, the name of this controller is confusing